sys/auto_init/auto_init.c

```diff
@@ -437,6 +437,42 @@ void auto_init(void)
     extern void auto_init_veml6070(void);
     auto_init_veml6070();
 #endif
+#ifdef MODULE_DS18
```

Comment:
seems like something went wrong when you rebased, most of these entries are unrelated to this PR, right?!

Reply:
Oops, I screwed up yep :(